pages/vote.tsx

```diff
@@ -102,7 +103,7 @@ class VotePage extends React.Component<VoteProps, VoteState> {
       })
     } else {
       if (!localStorage.getItem('ddd-voting-start-time')) {
-        localStorage.setItem('ddd-voting-start-time', dateTimeProvider.now().Value.toISOString())
+        localStorage.setItem('ddd-voting-start-time', moment().toISOString())
```

Comment:
When testing we have to change the date, which is then in the future so the backend call fails. This makes sure we always send now